index.html

```diff
@@ -1,179 +1,146 @@
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
     <link rel="stylesheet" href="styles/style.css" />
 
     <title>Bikes for Refugees</title>
-  </head>
+</head>
 
-  <body>
-    <div class="header">
+<body>
+    <header class="header">
 
         <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
+            <img src="./images/bikes-for-refugees_logo.jpg" alt="Logo for Bikes for Refugees Scotland" class="header__logo" />
         </a>
 
         <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
+            <ul class="navigation__list">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">Home</a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">About Us</a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">News & Events</a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">Contact Us</a>
+                </li>
+            </ul>
         </div>
 
-        <button>Donate Now</button>
+        <button class="button">Donate Now</button>
 
-    </div>
+    </header>
 
     <div class="content">
         <div class="main">
-          <div class="alert">
-            You've helped us give <span class="text-highlight">72</span> bikes
-            to refugees and asylum seekers so far. <strong>Thank you!</strong>
-          </div>
-
-          <div class="hero">
-            <h1>Bikes for Refugees</h1>
-            <p>
-              Providing donated bikes and accessories to refugees and asylum
-              seekers in Scotland.
-            </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
-          </div>
+            <div class="alert">
+                You've helped us give <span class="text-highlight">72</span> bikes to refugees and asylum seekers so far. <strong>Thank you!</strong>
+            </div>
 
-          <div>
-            <h2 class="heading-underline">Learn more</h2>
+            <div class="hero">
+                <h1 >Bikes for Refugees</h1>
+                <p class="hero h1">
```

Comment:
I think your hero image styling problems are coming from these classes in the <p> tag!